packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -50,31 +50,31 @@ export default function AnnualHighlightCards( {
 			</div>
 
 			<div className="highlight-cards-list">
-				<HighlightCard
+				<CountComparisonCard
```

Comment:
Would it make sense to pull the `heading` and `icon` values up a level and provide them when we are setting up the `count` values?

Currently we transform the API data in the caller and provide the `counts` configuration object. If we instead generated an array of card data objects (heading, icon, count) in the caller, we could avoid having configuration spread across two files. We could then map the passed in array here so this component would no longer be concerned about the number of cards being displayed.

Reply:
Yes, I think that'd be a good next step for refactoring the highlight cards 🙂